### PR TITLE
Feature delete

### PR DESCRIPTION
**Pull request by zitixiRose**
closing #27

### CODE REVIEW

**Review by mogeritz — COMMENTED** [review]
Looks good to me

**Review by mogeritz — APPROVED** [review]
Looks good

**Review by joyceweng — APPROVED** [review]

File: scripts/delete.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ * @param {number} id ID of the task to be removed from local storage
+ */
+export function deleteTaskById(id){
+
+    const taskList = getTasksFromStorage();
+    
+    //const del = document.getElementById('id');
+
+    for(var i = 0; i < taskList.length; i++) {
+        /**if(taskList[i] == del) {
```
Comment (melvyn9) [inline scripts/delete.js:18]: Please delete the commented out code, as mentioned by Prof it is a good habit to remove commented out code

File: scripts/delete.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+            taskList.splice(i, 1);
+        }**/
+
+        if(taskList[i].id == id){
```
Comment (melvyn9) [inline scripts/delete.js:22]: Please write some short comments about line 23, what are we splicing?

File: scripts/delete.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+            taskList.splice(i, 1);
+        }
+    }
+
```
Comment (melvyn9) [inline scripts/delete.js:26]: Some comments would be nice here, describing what line 27 and line 29 does

**Review by melvyn9 — COMMENTED** [review]
Overall great job! My only complaint is to add more comments to make code more readable.

File: scripts/delete.js (hunk 1 above)
Comment (vchou105) [reply scripts/delete.js:18]: I agree, unless we might use it in future (same for line 15)

File: scripts/delete.js (hunk 3 above)
Comment (vchou105) [reply scripts/delete.js:26]: I think more so for line 29 -- line 27 could be waived if we added that line 23 removes the corresponding task from the list (in my opinion)

**Pull request merged**